raiden/utils/typing.py

```diff
@@ -198,6 +198,9 @@ def typecheck(value: Any, expected: Union[Type, Tuple[Type, ...]]) -> None:
 T_RoomID = str
 RoomID = NewType("RoomID", T_RoomID)
 
+T_PeerCapabilities = Dict[str, Any]
```

Comment:
isnt it, at least in the current state `Dict[str, bool]` ?

Reply:
no, the spec defines at least `Dict[str, Union[bool, str]]`